### DISCUSSION TITLE
Stencil Toggle Expansion for Specific Group

### DISCUSSION THREAD

**Discussion by Fractal393** [clientIO/joint#2544] [Q&A]
### Introduction

I have been using the stencil with 2 groups, there is a toggle expansion button for all the groups next to the stencil header. I have disabled that using the groupsToggleButtons set to false which is the default behavior. Is there a way to individually disable toggle for groups? All i could find was the closed: Boolean property which sets the initial way the stencil is loaded.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
No, there is no option for that currently.

Do you need one group to be collapsible and the other not? Or do you need neither group to be collapsible?
